plasmapy/atomic/tests/test_atomic.py

```diff
@@ -328,21 +373,23 @@ def test_element_name(argument, expected):
 
 @pytest.mark.parametrize("argument, expected_error", element_name_error_table)
 def test_element_name_error(argument, expected_error):
-
+    """Test that element_name raises the expected exceptions."""
     with pytest.raises(expected_error):
         element_name(argument)
 
 
-def test_standard_atomic_weight_not_none():
-    assert standard_atomic_weight('N') is not None
-
-
 def test_standard_atomic_weight_value_between():
-    assert 30.973 < standard_atomic_weight('P').value < 30.974
+    """Test that standard_atomic_weight returns approximately the correct
+    value for phosphorus."""
+    assert 30.973 < standard_atomic_weight('P').value < 30.974, \
```

Comment:
Wait a minute, you can do three way inequalities now?! All that code I wrote, so ugly T.T

Reply:
This reminds me of the moment that I really began to appreciate the awesomeness of Python, which was when I saw the following code:
```Python
a, b = b, a
```